pingtest.sh

```diff
@@ -1,18 +1,31 @@
 #! /bin/bash
 
+<<<<<<< HEAD
```

Comment:
you've got a messy rebase here - notice you have both the new and the old version here between the `<<<<<` and `>>>>>`